Composer/packages/client/src/utils/luUtil.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { LuFile, DialogInfo, LuDiagnostic } from '@bfc/shared';
+import { Diagnostic, DiagnosticSeverity } from 'botbuilder-lg';
```

Comment:
We are importing `Diagnostic` from `lg` in `luutil`. Given the requirement of deep linking, in which we need a unified Diagnostic interface, can we take this change to define that in `indexer/shared` library, please refer to https://github.com/microsoft/BotFramework-Composer/pull/803/files#diff-437f284e7ac9b20ff3a724fcb6706a2c